google/cloud/vision/entity.py

```diff
@@ -55,7 +55,9 @@ def from_api_repr(cls, response):
         :rtype: :class:`~google.cloud.vision.entiy.EntityAnnotation`
         :returns: Instance of ``EntityAnnotation``.
         """
-        bounds = Bounds.from_api_repr(response['boundingPoly'])
+        bounds = []
+        if 'boundingPoly' in response:
+            bounds = Bounds.from_api_repr(response['boundingPoly'])
```

Comment:
So actually there will be more of this in the other PRs for other fields. Some API responses don't have the `boundingPoly` index.  Later on, `locale`, `mid`, `score` are returned optionally as well.

I used `response.get('locale', None)` for those however, so for this I could use 
```python
bounds = Bounds.from_api.repr(response.get('boundingPoly', []))
```